src/js/components/stock-movement-wizard/combined-shipments/SendMovementPage.jsx

```diff
@@ -471,14 +500,29 @@ class SendMovementPage extends Component {
    */
   submitStockMovement(values) {
```

Comment:
Maybe the name of the method should be changed to something like "sendFilesAndSave" because you've moved submitting to new "prepareRequestAndSave" method?